budgetportal/tests/test_posts_pages.py

```diff
@@ -15,6 +16,7 @@ def test_post_index_page(self):
         self.assertContains(response, self.post_index_page.title)
         self.assertContains(response, self.post_index_page.intro)
         self.assertContains(response, self.post_page.title)
+        self.breadcrumbs_test(response, self.post_index_page.get_ancestors())
```

Comment:
posts shouldn't have breadcrumbs at the moment

Reply:
ahh sorry I misunderstood - thought only posts cannot have it and the rule doesn't apply to post index ;-)